### PR TITLE
More information around operand and proxy images

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

https://enterprisedb.atlassian.net/browse/DOCS-198 

- How to select specific operand and proxy images for deployment.
- Reorganized installation topic to add more visual cues for steps.
- Cross-linked for more visibility among topics.

### CODE REVIEW

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: 'Identify your installation images and repositories'
+navigation:
+  - identify_repo_token
+  - identify_image_name
+  - assemble_command
+---
+
+Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade)
```
Comment (djw-m) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/index.mdx:9]:
```suggestion
If you plan on deploying a specific Postgres distribution or version, you will need to select the appropriate images and image versions.  Gather the following information before [installing EDB Postgres Distributed for Kubernetes](/postgres_distributed_for_kubernetes/latest/installation_upgrade)
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### Assembling the operand image name 
+
+These identifiers, together with their versions, translate to an operand image name. Here are a few examples: 
+
+| Postgres + version         | EDB Postgres Distributed + version | Build | Operand image name                       |
+|----------------------------|------------------------------------|-------|------------------------------------------|
+| EDB Postgres Advanced 16.2 | PGD 5.4.1                          | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
+| EDB Postgres Extended 15.6 | PGD 5.4.0                          | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
+| PostgreSQL 16.2            | PGD 5.4.1                          | 1     | postgresql-pgd:16.2-5.4.1-1              |
```
Comment (jsilvela) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:41]: What do the + signs mean in the headers?
And, given the example table, is there  a pointer to some place where the user can get a listing?
We devs don't have it. Just putting on my reader hat. If I don't see the version I want here, where do I go?

**Review by jsilvela — COMMENTED** [review]
Nice rework. I have a couple of questions.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+see [Installation](/postgres_distributed_for_kubernetes/latest/installation_upgrade/) for instructions on how add the repository, 
+deploy the images and create a certificate issuer. 
+
+## Example 
+
+The example command: 
+
+* Pulls images from the `k8s_enterprise_pgd` repository. 
+* Uses EBD Postgres Advanced Server 15.6.2 as a Postgres option. 
+* Uses PGD 5.4.1 as a Postgres Distributed version.
```
[the remark this answers is not in the document]
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/assemble_command.mdx:48]: Good point.
```suggestion
* Uses EBD Postgres Advanced Server 15.6.2 as the Postgres option. 
* Uses PGD 5.4.1 as the Postgres Distributed version.
```

File: product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx (hunk 2 above)
Comment (gvasquezvargas) [reply product_docs/docs/postgres_distributed_for_kubernetes/1/identify_images/identify_image_name.mdx:41]: >> What do the + signs mean in the headers?
I think I was trying to make clear that the image name is composed of the product name + version, but now that you point this out, I think it is kind of self-explanatory. If I get rid of the +, it still makes sense. 

```suggestion
| Postgres version         | EDB Postgres Distributed version | Build | Operand image name                       |
|----------------------------|------------------------------------|-------|------------------------------------------|
| EDB Postgres Advanced 16.2 | PGD 5.4.1                          | 1     | edb-postgres-advanced-pgd:16.2.0-5.4.1-1 |
| EDB Postgres Extended 15.6 | PGD 5.4.0                          | 1     | edb-postgres-extended-pgd:15.6-5.4.0-1   |
| PostgreSQL 16.2            | PGD 5.4.1                          | 1     | postgresql-pgd:16.2-5.4.1-1              |
```

>> Is there a pointer to some place where the user can get a listing?
I had the same question. Unfortunately, the listing can only be found on Cloudsmith, and not all customers have access to it 😞 . 
I considered manually adding a list, but that would be really hard to maintain as new versions and combinations are added.

File: product_docs/docs/postgres_distributed_for_kubernetes/1/installation_upgrade.mdx [hunk 4]
```diff
@@ -72,3 +88,5 @@ creating PGD clusters. See the
 If you're trying to install EDB Postgres Distributed for Kubernetes on Red Hat OpenShift,
 see [Red Hat OpenShift](openshift.md), which contains
 information on the certified operator maintained by EDB.
+import { startActivity } from "gatsby-cli/lib/reporter/redux/actions"
```
Comment (djw-m) [inline product_docs/docs/postgres_distributed_for_kubernetes/1/installation_upgrade.mdx:91]: I don't think you want this here do you? Looks like a fragment or accidental autocomplete.

**Review by djw-m — COMMENTED** [review]
LaGTM apart from the noted oddness in the review comment.

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**